src/App.jsx

```diff
@@ -1,35 +1,36 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import * as React from 'react';
 
-function App() {
-  const [count, setCount] = useState(0)
+const todoList = [
+  {
+    id: 1,
+    title: 'Read material for lesson 1.1'
+  },
+  {
+    id: 2,
+    title: 'Do textbook exercises'
+  },
+  {
+    id: 3,
+    title: 'Complete assignment'
+  }
```

Comment:
If it were me, I might have added these items inline to save space, e.g.:

{ id: 1, title: 'Read material for lesson 1.1' },
{ id: 2, title: 'Do textbook exercises' },
{ id: 3, title: 'Complete assignment' }

This is strictly a stylistic choice and there's nothing wrong with either option. However, when adding small items with just a few properties, using an inline style can help your code remain compact and readable. Just something to think about for the future! 🙂

Reply:
Thank you for the tip. I see your point.